CHANGELOG.md

```diff
@@ -235,13 +245,12 @@ Additions to existing modules
   rawModule          : RawModule R c ℓ
   ```
 
-* In `Algebra.Morphism.Structures`
+* In `Algebra.Morphism.Structures`:
   ```agda
-  module SuccessorSetMorphisms (N₁ : RawSuccessorSet a ℓ₁) (N₂ : RawSuccessorSet b ℓ₂) where
-    record IsSuccessorSetHomomorphism (⟦_⟧ : N₁.Carrier → N₂.Carrier) : Set _
-    record IsSuccessorSetMonomorphism (⟦_⟧ : N₁.Carrier → N₂.Carrier) : Set _
-    record IsSuccessorSetIsomorphism  (⟦_⟧ : N₁.Carrier → N₂.Carrier) : Set _
-
```

Comment:
I was afraid that would happen - I tried to do this via `git revert`, got into merge-hell over `CHANGELOG` and took in the simple-looking changes to this bit (because I couldn't believe that we wanted that complicated mess in the `CHANGELOG`...)  Will fix.

Reply:
Well, sorry for it being a mess...;-) wasn't sure how much detail was required... but it did get merged the first time around, so I guess it had been approved...